persistence/persistence-mybatis/src/main/java/org/cbioportal/persistence/mybatis/util/CacheMapUtil.java

```diff
@@ -89,11 +89,17 @@
     }
 
     @PostConstruct
-    private void initializeCacheMemory() {
+    private void init() {
+        initializeCacheMemory();
+    }
+
+    public void initializeCacheMemory() {
+        
         // CHANGES TO THIS LIST MUST BE PROPAGATED TO 'GlobalProperties'
```

Comment:
To what list?

Reply:
This is not my comment. And I also did not make changed to that list :) So, I considered this to be not relevant to me.